pom.xml

```diff
@@ -70,7 +64,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.aws-java-sdk</groupId>
             <artifactId>aws-java-sdk-minimal</artifactId>
-            <version>1.12.215-339.vdc07efc5320c</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>joda-time</groupId>
+                    <artifactId>joda-time</artifactId>
+                </exclusion>
```

Comment:
Not sure this is right. From `artifact-manager-s3` it is then necessary to readd this dep. In the `aws-java-sdk-minimal` plugin

```
[WARNING] Bundling transitive dependency joda-time-2.8.1.jar (via aws-java-sdk-core)
```

Perhaps `artifact-manager-s3` just needs to _directly_ declare a dep on that plugin.